FAQ.md

```diff
@@ -0,0 +1,62 @@
+# FAQs 
+_From emailed responses_
+
+**Q:  Will it be kid friendly?**
+
+A:  Yes! Children under the age of 10 can come for free.  We will have a family camping area, where you can pitch your tent away from the music and late night gatherings. We will also have a family area with activities that young kids can enjoy--hands on creativity.
+
+Many of the activities will also appeal to all ages: from harvesting food for our meals, hiking up the local stream, going to the beach, to live music jams and yoga.
+
+However, you will be expected to be responsible for your children at all times. We won’t have babysitting and you can’t just drop your kids off in the family activities tent. So if you imagine yourself fully participating in workshops, parties, hack-a-thons and more, I think you should plan on bringing someone whose main focus is watching the kids.
+
+**Q:  Will you have a tent big enough for 2 adults, 2 kids, and baby? Or hostel or RV (if able to rent RV)**
+
+A:  Good question!  Our [Glamping tents](https://archive.org) come with one queen bed or two twin cots, so I think it would be a tight squeeze.  Each tent is $400 for the 3-4 nights.
+
+You could rent a room at the nearby Costanoa Lodge, which also has luxury glamping tents.
+The rooms at the nearby Pigeon Point Youth Hostel are already booked up, but you could check for cancellations.
+I think your best bet is to rent an RV in San Francisco (there is also an Air B-n-B for RVs..)  
+We will have spaces for RVs for a nominal RV parking charge.  Prices range from $100/night to $300+ for RVs that sleep 6. 
```

Comment:
Unclear whether RV option is booked through DWeb Camp or independently arranged.

Reply:
I fixed that